main.go

```diff
@@ -273,6 +276,18 @@ func (o *Options) Run() error { //nolint:gocyclo
 		}
 	}
 
+	timeNow := time.Now()
+
+	if len(o.PinnedDateTime) > 0 {
+		timeNow, err = time.Parse(time.RFC3339, o.PinnedDateTime)
+
+		if err != nil {
+			log.Errorf("Error parsing pinnedDateTime: %#v", err)
```

Comment:
Does Errorf exit? I don't think it does, so it might be a little surprising to a user who provides an invalid pinned date but sippy starts without it.

Reply:
No, but I can make it a hard error